apps/opik-frontend/src/components/shared/LoadableSelectBox/LoadableSelectBox.tsx

```diff
@@ -270,52 +270,80 @@ export const LoadableSelectBox = ({
-                      checked={isSelected(option.value)}
-                      className="shrink-0"
-                    />
-                  ) : (
-                    <div className="min-w-4">
-                      {isSelected(option.value) && (
-                        <Check className="size-3.5 shrink-0" strokeWidth="3" />
+              {filteredOptions.map((option) => {
+                return (
+                  <div
+                    key={option.value}
+                    className={cn(
+                      "group flex cursor-pointer items-center gap-2 rounded-md px-4 hover:bg-primary-foreground",
+                      option.description ? "min-h-12 py-2" : "h-10",
+                    )}
+                    onClick={() => {
+                      if (multiselect) {
+                        const newSelectedValues = isSelected(option.value)
+                          ? selectedValues.filter((v) => v !== option.value)
+                          : [...selectedValues, option.value];
+                        onChange &&
+                          (onChange as (value: string[]) => void)(
+                            newSelectedValues,
+                          );
+                      } else {
+                        onChange &&
+                          (onChange as (value: string) => void)(option.value);
+                        openChangeHandler(false);
+                      }
+                    }}
+                  >
+                    {multiselect ? (
+                      <Checkbox
+                        checked={isSelected(option.value)}
+                        className="shrink-0"
+                      />
+                    ) : (
+                      <div className="min-w-4">
+                        {isSelected(option.value) && (
+                          <Check
+                            className="size-3.5 shrink-0"
+                            strokeWidth="3"
+                          />
+                        )}
+                      </div>
+                    )}
+
+                    <div className="min-w-0 flex-1">
+                      <div className="comet-body-s truncate">
+                        {option.label}
+                      </div>
+                      {option.description && (
+                        <div className="comet-body-xs truncate text-muted-foreground">
+                          {option.description}
+                        </div>
                       )}
                     </div>
-                  )}
 
-                  <div className="min-w-0 flex-1">
-                    <div className="comet-body-s truncate">{option.label}</div>
-                    {option.description && (
-                      <div className="comet-body-xs truncate text-muted-foreground">
-                        {option.description}
-                      </div>
+                    {option.action && (
+                      <TooltipWrapper content="Open in a new tab">
```

Comment:
Is the tooltip working in both places? In video, saw only 1

Let's extract it to a global const too

Don't remember we have it in other Navigations... did work on NavigationTag component, but believe they are separate

Reply:
Yep, in two places
<img width="464" height="433" alt="image" src="https://github.com/user-attachments/assets/5aa22647-6ed3-4085-9a96-1ba2161459dd" />